notification-server/Cargo.toml

```diff
@@ -1,11 +1,30 @@
 [package]
```

Comment:
These are not all up to date.

I just code referenced kpi-tracker and hence did not want to update all the different versions. 

I can create a task on updating when everything else is working.

Reply:
I suggest starting with up-to-date packages if possible. Refactoring later might become a big task and our newer libraries tend to have better support for many implementation flows so give them a go, please :).

Reply:
I am dependent on bumping tonic to 0.11 on the concordium_sdk before I can bump it here. I will not consider that a blocker for this PR. https://github.com/Concordium/concordium-rust-sdk/blob/main/Cargo.toml